bin/commands/transform.js

```diff
@@ -0,0 +1,73 @@
+// Node.js core.
+const { join } = require("path");
+
+// jscodeshift engine
+const jscodeshift = require("jscodeshift/dist/Runner");
+
+// Enquirer engine.
+const { prompt } = require("enquirer");
+
+/**
+ * Prompt's configuration
+ * choices array value have to be name of transform file
+ */
+const promptOptions = [
+  {
+    type: "select",
+    name: "type",
+    message: "What kind of transformation do you want to perform?",
+    choices: [
+      { name: "find -> findMany", value: "change-find-to-findMany" },
+      {
+        name: "strapi-some-package -> @strapi/some-package",
+        value: "update-strapi-scoped-imports",
+      },
+      {
+        name: ".models -> .contentTypes",
+        value: "change-model-getters-to-content-types",
+      },
+      {
+        name: "strapi.plugins['some-plugin'] -> strapi.plugin('some-plugin')",
+        value: "use-plugin-getters",
+      },
+      {
+        name: "strapi.plugin('some-plugin').controllers['some-controller'] -> strapi.plugin('some-plugin').controller('some-controller')",
+        value: "update-top-level-plugin-getter",
+      },
+      {
+        name: "Add arrow function for service export",
+        value: "use-arrow-function-for-service-export",
+      },
+      {
+        name: "Add strapi to bootstrap function params",
+        value: "add-strapi-to-bootstrap-params",
+      },
+    ],
+    result() {
+      return this.focused.value;
+    },
+  },
+  {
+    type: "input",
+    name: "path",
+    message: "Enter the path to file(s) or folder to transform",
```

Comment:
What do you think about making this prompt more specific based on the transformation selected?  For example if `add-strapi-to-bootstrap-params` was selected, prompt: `Enter the path to the bootstrap.js file`

Reply:
I think the only two transforms that we have right now that would need something like this are:

`add-strapi-to-bootstrap-params` ⇒ needs a path to the bootstrap.js file

`use-arrow-function-for-service-export` ⇒ needs a path to a folder containing multiple services or to a specific service file

Unless you see something else.  Also if you would like to arrange a time to discuss further on discord we could arrange that next week.

Reply:
Let's discuss on discord so we could talk about it in details ;)